source/_integrations/ohme.markdown

```diff
@@ -17,7 +17,7 @@ ha_platforms:
   - sensor
   - switch
   - time
-ha_quality_scale: silver
+ha_quality_scale: gold
```

Comment:
This is handled automatically and lead by the core codebase. We shouldn't change this in this pull request.